### DISCUSSION TITLE
Getting "Error: Parse Error: Expected HTTP/" when trying to query admin api on Windows

### DISCUSSION THREAD

**Discussion by olegesan** [ory/hydra#2639] [Q&A] [answered]
Hey Hydra community!
I am running hydra in Docker on a Windows machine. I face the issue that when I make any kind of request to Admin api (localhost:4445) I get an error. If I do it in postman it logs "Error: Parse Error: Expected HTTP/". If I make the same request through curl in bash, it logs "curl: (1) Received HTTP/0.9 when not allowed". 
This is weird considering that when I make a request to localhost:4444 it returns no error. As a test I made requests to health/alive and health/read. Another weird observation is that hydra is not even logging anything like the request has never been made (with successful request it produces a log entry)
I.e.
`curl --location --request GET 'http://localhost:4445/health/alive'`
would return 
`curl: (1) Received HTTP/0.9 when not allowed`
The error logged by nodeJS is 
`Parse Error: Expected HTTP/`
However, if I run a curl request from wsl terminal, it behaves as expected for both endpoints. What could be the explanation/solution for this?
I.e.
`curl --location --request GET 'http://localhost:4445/health/alive'`
would respond with
`{"status":"ok"}`
I tried researching if anybody had the same issue or if there were special notes for windows. Could not find anything. 
Updated hydra version to 1.10.3.

**Comment by vinckr** [comment, marked as answer]
I think this is more of an issue with Docker, curl, or wsl in Windows, rather than something in Hydra. 

I did run the Hydra quickstart recently on Windows and it worked fine, happy to see if I can reproduce your issue on my machine if you don't find a solution. 
Did you write your own hydra.yaml, and if so could you post it, possibly with instructions on how I could reproduce it?

For curl I found:
> It seems that Windows curl doesn’t like single quotes. This works:
> `curl -g -H "Content-type: application/json" -X POST -d "{\"text\":\"Patrick This is a line of line of text.\"}"`
> Putting every string in double quotes and escape any double quotes within with a \.
> And it needs the header line.

**Reply by vinckr** [reply]
Hey @olegesan , just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
